app/Api/Internal/Requests/BaseJsonApiRequest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Api\Internal\Requests;
+
+use Illuminate\Contracts\Validation\Validator;
+use Illuminate\Foundation\Http\FormRequest;
+use LaravelJsonApi\Core\Exceptions\JsonApiException;
+
+abstract class BaseJsonApiRequest extends FormRequest
+{
+    /**
+     * Handle a failed validation attempt for JSON:API requests.
+     * This matches the error structure used in Handler.php for JsonApiException.
+     */
+    protected function failedValidation(Validator $validator): void
+    {
+        throw JsonApiException::error([
+            'status' => '422',
```

Comment:
Should we update the connect API to return 422 for invalid/missing parameter errors? It's currently returning 400. This seems more appropriate, and I don't think anything actually relies on the http status code - rcheevos doesn't even look at it unless the body is empty.

Reply:
I agree that a 422 seems more appropriate. I haven't studied what it would take to have the Connect API return 422 responses for validation errors. Is it a single place that'd need to be updated, or would each routine need to have a surgical modification?

Reply:
There's a couple places in BaseApiAction that would need to be updated. I'll address it the next time I convert an action.